cypress/integration/search/post_search_display.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+describe('Post search display', () => {
+    it('S14252 After clearing search query, search options display', () => {
+        // # Login and navigate to the app
+        cy.apiLogin('user-1');
+        cy.visit('/');
+        const searchWord = 'Hello';
+
+        // # post message
+        cy.postMessage('Hello');
+
+        // # search word in searchbox
+        cy.get('#searchBox').type(searchWord).type('{enter}');
+
+        // # searchbox should have attribute 'value'
+        cy.get('#searchBox').should('have.attr', 'value', searchWord);
```

Comment:
You should be able to combine this assertion with the above command. In addition, you can also use `should('have.value', searchWord)` vs the `have.attr` way.